src/azul/service/manifest_service.py

```diff
@@ -90,7 +90,12 @@ def get_manifest(self, format_: str, filters: Filters) -> Tuple[str, bool]:
         manifest was reused and False if a new manifest was generated.
         """
         generator = ManifestGenerator.for_format(format_, self, filters)
-        object_key, presigned_url = self._get_cached_manifest(generator, format_, filters)
+
+        # Passing 'object_key' as a parameter to `get_manifest()` indicates a
+        # cached manifest was not available, it must be generated.
+        object_key, presigned_url = self._get_cached_manifest(generator,
+                                                              format_,
+                                                              filters) if object_key is None else [object_key, None]
```

Comment:
This is a bit insidious. I had to look at it three time before I realized that the _get_cached_manifest call is conditional.

A if statement would be better.